modules/core/04-channel/keeper/packet_test.go

```diff
@@ -217,6 +217,43 @@ func (suite *KeeperTestSuite) TestSendPacket() {
 
 			channelCap = capabilitytypes.NewCapability(5)
 		}, false},
+		{
+			"invalid flush status: FLUSHING",
+			func() {
+				suite.coordinator.Setup(path)
+				sourceChannel = path.EndpointA.ChannelID
+
+				channel := path.EndpointA.GetChannel()
+				channel.FlushStatus = types.FLUSHING
+				path.EndpointA.SetChannel(channel)
+			},
+			false,
+		},
+		{
+			"invalid flush status: FLUSHCOMPLETE",
+			func() {
+				suite.coordinator.Setup(path)
+				sourceChannel = path.EndpointA.ChannelID
+
+				channel := path.EndpointA.GetChannel()
+				channel.FlushStatus = types.FLUSHCOMPLETE
+				path.EndpointA.SetChannel(channel)
+			},
+			false,
+		},
+		{
+			"channel is in INITUPGRADE stage",
+			func() {
+				suite.coordinator.Setup(path)
+
+				path.EndpointA.ChannelConfig.ProposedUpgrade.Fields.Version = ibcmock.UpgradeVersion
+				path.EndpointB.ChannelConfig.ProposedUpgrade.Fields.Version = ibcmock.UpgradeVersion
```

Comment:
is this line necessary?

Reply:
it's not great catch! We can remove this.